src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1595,7 +1595,7 @@ internal enum ErrorCode
         ERR_NoConvToIDispWrongAsync = 8418,
 
         #region diagnostics introduced for recursive patterns
-        ERR_InferredRecursivePatternType = 8501,
+        // ERR_InferredRecursivePatternType = 8501, // unused
```

Comment:
>unused [](start = 55, length = 6)

It is not clear whether this error number can be reused. If that is the case, please mark it as "available" instead.